libs/domains/variables/feature/src/lib/variables-action-toolbar/variables-action-toolbar.tsx

```diff
@@ -0,0 +1,67 @@
+import { ActionToolbar, DropdownMenu, Icon, Tooltip } from '@qovery/shared/ui'
+
+export interface VariablesActionToolbarProps {
+  onImportEnvFile: () => void
+  onCreateVariable: () => void
+  onCreateVariableFile: () => void
+}
+
+export function VariablesActionToolbar({
+  onImportEnvFile,
+  onCreateVariable,
+  onCreateVariableFile,
+}: VariablesActionToolbarProps) {
+  return (
+    <ActionToolbar.Root>
+      <DropdownMenu.Root>
+        <DropdownMenu.Trigger asChild>
+          <ActionToolbar.Button color="brand" variant="solid" className="justify-center border-r border-brand-400">
+            <Icon iconName="ellipsis-v" />
+          </ActionToolbar.Button>
+        </DropdownMenu.Trigger>
+        <DropdownMenu.Content align="end">
+          <DropdownMenu.Item onClick={onImportEnvFile}>
```

Comment:
You need to remove icon children and use directly `icon` props, it avoid to have difference color between icon for all `DropdownMenu` 
<img width="405" alt="Capture d’écran 2024-03-06 à 13 34 29" src="https://github.com/Qovery/console/assets/533928/df31fbf6-2da0-4d52-8ec4-1bc550b3948e">
<img width="269" alt="Capture d’écran 2024-03-06 à 13 34 45" src="https://github.com/Qovery/console/assets/533928/a75f1364-4eb0-4d7b-b83b-0d7e8b5578c2">


```suggestion
          <DropdownMenu.Item onClick={onImportEnvFile} icon={<Icon iconName="cloud-upload" />}>
```

Reply:
This was intentional to preserve icon color but I agree we should align design.
So I've updated the icons and the storybook stories
https://github.com/Qovery/console/pull/1227/commits/20e8b257ca399589682b2136922530e819f500c0
https://github.com/Qovery/console/pull/1227/commits/0f01d0d396de4ad4d6f4d1731830a625ec7807f2